plantuml-mode.el

```diff
@@ -165,15 +176,15 @@
 (defun plantuml-set-exec-mode (mode)
   "Set the execution mode MODE for PlantUML."
   (interactive (let* ((completion-ignore-case t)
-                      (supported-modes        '("jar" "server")))
-                 (list (completing-read (format "Exec mode [%s]: " plantuml-exec-mode)
```

Comment:
we actually need to return a `list` to `interactive` otherwise we get back into the situation we had with #97